front/app/containers/EventsShowPage/components/ProjectLink.tsx

```diff
@@ -20,10 +26,17 @@ const ProjectLink = ({
   projectSlug,
 }: ProjectLinkProps) => {
   const { formatMessage } = useIntl();
+  const isMobileOrSmaller = useBreakpoint('phone');
```

Comment:
```suggestion
  const isPhoneOrSmaller = useBreakpoint('phone');
```

From my understanding, mobile = phone + tablet :)